### PR TITLE
Feat/footer

### PR DESCRIPTION
**Pull request by 99mac99**

### CODE REVIEW

File: components/Footer/PartnersBox.tsx [hunk 1]
```diff
@@ -1,18 +1,45 @@
...
+			}}
+		>
+			<Box sx={{ margin: '20px' }}>
+				<Link
+					href='https://unimind-dao.com/'
+					underline='none'
+					sx={{ color: 'white' }}
+				>
+					Contact
+				</Link>
```
Comment (TomaszKotowski) [inline components/Footer/PartnersBox.tsx]: Use `CustomLink`

File: components/Footer/PartnersBox.tsx [hunk 2]
```diff
@@ -1,18 +1,45 @@
...
+			</Box>
+			<Box sx={{ margin: '20px' }}>
+				<Link
+					href='https://twitter.com/UnimindDAO'
+					sx={{ color: 'white' }}
+					underline='none'
+				>
+					Webtrzy.xyz
+				</Link>
```
Comment (TomaszKotowski) [inline components/Footer/PartnersBox.tsx]: Use `CustomLink`

File: components/Footer/PartnersBox.tsx [hunk 3]
```diff
@@ -1,18 +1,45 @@
...
+			</Box>
+			<Box sx={{ margin: '20px' }}>
+				<Link
+					href='https://nftpolska.net/'
+					sx={{ color: 'white' }}
+					underline='none'
+				>
+					NFTPolska
+				</Link>
```
Comment (TomaszKotowski) [inline components/Footer/PartnersBox.tsx]: Use `CustomLink`

File: components/Footer/PartnersBox.tsx [hunk 4]
```diff
@@ -1,18 +1,45 @@
...
\ No newline at end of file
+	return (
+		<Box
+			sx={{
+				width: '100%',
+				display: 'flex',
+				flexDirection: 'row',
+				justifyContent: 'center',
+				alignItems: 'center',
+				color: 'white',
```
Comment (TomaszKotowski) [inline components/Footer/PartnersBox.tsx]: Export `sx` to variable if we did the same in another file for consistency.

**Review by TomaszKotowski — CHANGES_REQUESTED** [review]

File: components/CustomLink/CustomLink.tsx [hunk 5]
```diff
@@ -1,7 +1,7 @@
-import { Link, LinkProps } from "@mui/material";
+import { Link, LinkProps } from '@mui/material';
 
 export const CustomLink = (props: LinkProps) => (
-  <Link underline="none" color="black" rel="noopener noreferrer" {...props}>
-    {props.children}
-  </Link>
+	<Link underline='none' color='black' rel='noopener noreferrer' {...props}>
```
Comment (adiusz) [inline components/CustomLink/CustomLink.tsx]: aren't double apostrophes better for JSX/HTML? Also, are you using 8 spaces? xD

File: components/Footer/EndBox.tsx [hunk 6]
```diff
@@ -1,9 +1,18 @@
-import styles from './Footer.module.css';
+import { Box} from '@mui/system';
 
 export default function Footer() {
+	const sx = {
+		width: '100%',
+		display: 'flex',
+		flexDirection: 'row',
```
Comment (adiusz) [inline components/Footer/EndBox.tsx:7]: as agreed — we should use styled-components

File: components/Footer/PartnersBox.tsx [hunk 7]
```diff
@@ -1,18 +1,45 @@
...
+			}}
+		>
+			<Box sx={{ margin: '20px' }}>
```
Comment (adiusz) [inline components/Footer/PartnersBox.tsx]: you can put this data in the array here and use map method to render these items

**Review by adiusz — CHANGES_REQUESTED** [review]

**Review by adiusz — CHANGES_REQUESTED** [review]
what about my comments from a month ago?

**Comment by 99mac99** [comment]
Take a look at the commits xD

**Pull request merged**